### PR TITLE
match some functions in kmr_20/8EBE50.c

### PR DESCRIPTION
**Pull request by fewffwa**

### CODE REVIEW

**Comment by BowserSlug** [comment]
⚠️ This PR introduces 5 warnings:
- src/world/area_kmr/kmr_20/8EBE50.c:10: warning: return-type defaults to `int'
- src/world/area_kmr/kmr_20/8EBE50.c:11: warning: implicit declaration of function `func_8005566C'
- src/world/area_kmr/kmr_20/8EBE50.c:15: warning: return-type defaults to `int'
- src/world/area_kmr/kmr_20/8EBE50.c:30: warning: return-type defaults to `int'
- src/world/area_kmr/kmr_20/8EBE50.c:5: warning: return-type defaults to `int'

File: src/world/area_kmr/kmr_20/8EBE50.c [hunk 1]
```diff
@@ -2,11 +2,20 @@
 
 #include "world/common/atomic/Pipe.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8EBE50", func_80240310_8EC130);
+ func_80240310_8EC130(void) {
+    set_map_transition_effect(4);
+    return 2;
+}
```
Comment (nanaian) [inline src/world/area_kmr/kmr_20/8EBE50.c:8]: Please make this change to all 4 funcs
```suggestion
ApiStatus func_80240310_8EC130(Evt* script, s32 isInitialCall) {
    set_map_transition_effect(4);
    return ApiStatus_DONE2;
}
```

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/world/area_kmr/kmr_20/8EBE50.c [hunk 2]
```diff
@@ -2,11 +2,20 @@
...
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8EBE50", func_80240330_8EC150);
+ func_80240330_8EC150(void) {
+    func_8005566C(0, 0x3E8, 0x3F);
```
Comment (nanaian) [inline src/world/area_kmr/kmr_20/8EBE50.c:11]: Can you declare this function in functions.h? i.e. add `void func_8005566C(s32, s32, s32);` to it

**Review by ethteck — APPROVED** [review]
Feel free to merge after alex's changes are addressed. The 0x3E8 would probably be better as 1000 as well

**Pull request merged**